app/src/main/java/com/example/ncov19traking/ui/graphs/GraphsFragment.kt

```diff
@@ -30,11 +31,14 @@ class GraphsFragment : Fragment() {
     ): View? {
         val root = inflater.inflate(R.layout.fragment_notifications, container, false)
         val chart: LineChart = root.findViewById(R.id.chart)
+        val progressBar: ProgressBar = root.findViewById(R.id.graph_fragment_progressBar)
         val textColor = ContextCompat.getColor(root.context, R.color.graphTextColor)
         setUpChart(chart, textColor)
+        chart.visibility = View.GONE
         graphsViewModel.nCoVAllHistoricalData.observe(viewLifecycleOwner, Observer { nCovTimeline ->
-            val allLineData = ArrayList<ILineDataSet>()
+            progressBar.visibility = View.VISIBLE
```

Comment:
KTX has a method for this `isVisible` https://android.github.io/android-ktx/core-ktx/androidx.view/android.view.-view/is-visible.html , check it out.

Reply:
KTX is something that I enjoy to learn, fixed